mindsdb_native/libs/helpers/text_helpers.py

```diff
@@ -10,9 +10,38 @@
 """
 
 from mindsdb_native.libs.constants.mindsdb import *
+from collections import Counter
+import string
 import json
 import hashlib
 import numpy
+import flair
+
+
+def analyze_sentences(data):
+    """
+    :param data: list of str
+    :returns: 
+    tuple(
+        int: nr words total,
+        tuple(
+            dict: word_dist
+            dict: nr_words_dist
+        )
+    )
+    """
+    nr_words = 0
+    word_dist = Counter()
```

Comment:
Why make this a `Counter ?

Reply:
should be `defaultdict(lambda: 0),` fixed